src-cpp/rdkafkacpp.h

```diff
@@ -1830,6 +1833,7 @@ class RD_EXPORT Headers {
       err_ = other.err_;
       value_size_ = other.value_size_;
 
+      free(value_);
```

Comment:
```
if (value_)
  free(value_);
```

Reply:
free(nullptr) is defined, but i can change this.